ui/src/translations/es.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  'GlobalHome.howDoesItWork': '¿Como funciona?',
```

Comment:
Add a comment saying these are temporary translations for testing.

Reply:
Done.